src/translations/en.json

```diff
@@ -41,28 +41,28 @@
         "on": "Wet"
       },
       "gas": {
-        "off": "Clear",
-        "on": "Detected"
+        "off": "Nothing",
+        "on": "Gas detected"
       },
       "motion": {
-        "off": "[%key:state::binary_sensor::gas::off%]",
-        "on": "[%key:state::binary_sensor::gas::on%]"
+        "off": "Nothing",
+        "on": "Motion detected"
       },
       "occupancy": {
-        "off": "[%key:state::binary_sensor::gas::off%]",
-        "on": "[%key:state::binary_sensor::gas::on%]"
+        "off": "Free",
```

Comment:
I'd prefer Empty/Occupied here.

Reply:
The problem is that we need to have it fix in a badge, which means like 5/6 characters.

Reply:
Binay sensor state text is actually only shown in the more-info, at least currently.

Reply:
Ah, good point.

Reply:
Empty sounds good